modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -949,7 +949,7 @@ protected function register_controls() {
 				'name' => 'box_background_color',
 				'types' => [ 'classic', 'gradient' ],
 				'exclude' => [ 'image' ],
-				'selector' => "{$nested_tabs_content_selector_class}",
+				'selector' => $nested_tabs_content_selector_class,
```

Comment:
I know we have used this `{$..}` throughout the whole php file. I am thinking can you check if we can remove it everywhere, for confirmity sake?

Reply:
It does seem redundant in this case, where there is nothing else being processed inside the double quotes.
I will do as you said and change it in the whole file.

Reply:
I've checked and there are no more unnecessary occurrences :)